crowbar_framework/lib/product.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2015, SUSE LINUX Products GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  class Product
+    def self.is_ses?
+      BarclampCatalog.barclamps.key?("suse-enterprise-storage")
+    end
+  end
+end
+
```

Comment:
When Thomas suggested moving this to Crowbar module, he perhaps forgot to mention that it requires moving the file as well (to make auto-loading work). This is why the travis build fails now. Could you please try moving the file under `./lib/crowbar/`? You can also run `bundle exec rake spec` at any time to run the tests locally.